torch/nn/modules/activation.py

```diff
@@ -106,10 +108,9 @@ def extra_repr(self) -> str:
 
 
 class RReLU(Module):
-    r"""Applies the randomized leaky rectified linear unit function, element-wise,
-    as described in the paper:
+    r"""Applies the randomized leaky rectified linear unit function, element-wise.
 
-    `Empirical Evaluation of Rectified Activations in Convolutional Network`_.
+    Method described in the paper:`Empirical Evaluation of Rectified Activations in Convolutional Network`_.
```

Comment:
nit: not your fault but looks like this is missing the url of the paper https://arxiv.org/abs/1505.00853, could you add it please

Reply:
This is my bad, I just looked at the rendered public documentation on main and it seems that the paper is linked properly, so there's actually no need to change anything here

Reply:
Deleted second mention of research paper.